.github/workflows/ci.yml

```diff
@@ -228,12 +229,18 @@ jobs:
           # gnupg2 is for signing the resulting binary.
           install: |
             apt-get update -q -y
-            apt-get install -q -y build-essential libz-dev git python3-pip gnupg2 curl
+            apt-get install -q -y build-essential libz-dev git python3-pip gnupg2 curl zip
+            touch /tmp/github_env
```

Comment:
Just to make sure I create the dummy file first.
Also zip [is needed](https://github.com/VirtusLab/scala-cli/blob/8ba8022069b2f35ff6374a8aeeb8f252cbc190cd/.github/scripts/generate-os-packages.sh#L129) inside the vm